lib/internal/Magento/Framework/View/Asset/Minification.php

```diff
@@ -143,12 +143,26 @@ public function getExcludes($contentType)
         if (!isset($this->configCache[self::XML_PATH_MINIFICATION_EXCLUDES][$contentType])) {
             $this->configCache[self::XML_PATH_MINIFICATION_EXCLUDES][$contentType] = [];
             $key = sprintf(self::XML_PATH_MINIFICATION_EXCLUDES, $contentType);
-            foreach (explode("\n", $this->scopeConfig->getValue($key, $this->scope)) as $exclude) {
+            $excludeValues = $this->getMinificationExcludeValues($key);
+            foreach ($excludeValues as $exclude) {
                 if (trim($exclude) != '') {
                     $this->configCache[self::XML_PATH_MINIFICATION_EXCLUDES][$contentType][] = trim($exclude);
                 }
             }
         }
         return $this->configCache[self::XML_PATH_MINIFICATION_EXCLUDES][$contentType];
     }
+
+    /**
+     * Get minification exclude values from configuration
+     *
+     * @param string $key
+     * @return string[]
+     */
+    private function getMinificationExcludeValues($key)
+    {
+        $configValues = $this->scopeConfig->getValue($key, $this->scope) ?? [];
```

Comment:
Hi @alexeya-ven 

`getValue` doesn't always return a string, the return value is being defined as [`mixed`](https://github.com/magento/magento2/blob/231709b/lib/internal/Magento/Framework/App/Config/ScopeConfigInterface.php#L27)

In this case it returns an array instead of a string, because an extra level of nodes was added in the xml tree. So it's no longer a long string which could consist of multiple lines. But zero, one or multiple xml nodes instead.
By then using `array_values`, I'm getting the contents of those xml nodes which gives us an array of strings again, like the initial implementation did (which used `explode`).

The idea was to make this configuration mergeable so you can add multiple exclusions from different modules, which wasn't possible before, one module overwrote the configuration of another module.

I hope this makes it a bit more clear?